spec/spec_helper.rb

```diff
@@ -9,14 +9,8 @@
 # Require simplecov before loading ..dummy/config/environment.rb because it will cause metasploit-credential/lib to
 # be loaded, which would result in Coverage not recording hits for any of the files.
 require 'simplecov'
-require 'coveralls'
 
-if ENV['TRAVIS'] == 'true'
-  # don't generate local report as it is inaccessible on travis-ci, which is why coveralls is being used.
```

Comment:
Interestingly github can artifact files, so we could extract these reports with github actions in the future

Reply:
This is done now, resolving https://github.com/rapid7/metasploit-credential/pull/152/files#r622038016